xdsl/traits.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 from dataclasses import dataclass, field
+from dataclasses import dataclass
```

Comment:
Interesting, do double imports not trigger pyright?

Reply:
Apparently not! Will remove it, good catch!